cdc/sink/dispatcher/event_router.go

```diff
@@ -107,23 +118,58 @@ func NewEventRouter(cfg *config.ReplicaConfig, defaultTopic string) (*EventRoute
 	}, nil
 }
 
-// GetTopic returns the target topic.
-func (s *EventRouter) GetTopic(row *model.RowChangedEvent) string {
+// GetTopicForRowChange returns the target topic for row changes.
+func (s *EventRouter) GetTopicForRowChange(row *model.RowChangedEvent) string {
 	topicDispatcher, _ := s.matchDispatcher(row.Table.Schema, row.Table.Table)
-	return topicDispatcher.DispatchRowChangedEvent(row)
+	return topicDispatcher.Substitute(row.Table.Schema, row.Table.Table)
 }
 
-// GetPartition returns the target partition.
-func (s *EventRouter) GetPartition(
+// GetTopicForDDL returns the target topic for DDL.
+// FIXME: Now we can't handle rename tables because
+// we are missing the old and new tables information.
+func (s *EventRouter) GetTopicForDDL(ddl *model.DDLEvent) string {
+	var schema, table string
+	if ddl.PreTableInfo != nil {
+		if ddl.PreTableInfo.Table == "" {
+			return s.defaultTopic
+		}
+		schema = ddl.PreTableInfo.Schema
+		table = ddl.PreTableInfo.Table
```

Comment:
If `table` information is nil, should it match the rule `schema.*`

Reply:
Do you mean a situation where there is neither a new information nor an old information? Does this happen with the ddl supported by cdc?

Reply:
When the table does not exist, we consider it to be a schema-level DDL, which should be sent to the default topic according to the spec.